### PR TITLE
bin/build-cache-lxd: minio download and copy into each go version's bin dir

### PR DESCRIPTION
**Pull request by tomponline**
Signed-off-by: Thomas Parrott <thomas.parrott@canonical.com>

### CODE REVIEW

File: bin/build-cache-lxd [hunk 1]
```diff
@@ -129,9 +128,12 @@ for version in 1.13 1.17 1.18 tip; do
         fi
     done
 
+    cp "${GOPATH}/minio" "${GOPATH}/bin/minio"
```
Comment (stgraber) [inline bin/build-cache-lxd:131]: Same thing here, this should be conditional on minio existing.

Comment (stgraber) [reply bin/build-cache-lxd:131]: I think I'd remove the `chmod +x` from above and have this block do:
```
if [ -e "${GOPATH}/minio" ]; then
    cp "${GOPATH}/minio" "${GOPATH}/bin/minio"
    chmod +x "${GOPATH}/bin/minio"
fi
```

File: bin/build-cache-lxd [hunk 2]
```diff
@@ -129,9 +126,18 @@ for version in 1.13 1.17 1.18 tip; do
...
+        cp "${GOPATH}/minio" "${GOPATH}/bin/minio"
+        chmod +x "${GOPATH}/bin/minio"
+    fi
+
     mv "${GOPATH}/bin" "${GOPATH}/bin.$(go version | cut -d' ' -f3)"
 done
 
+if [ -f "${GOPATH}/minio" ]; then
+    rm "${GOPATH}/minio"
+fi
```
Comment (stgraber) [inline bin/build-cache-lxd:139]: `rm -f` would have been easier :)

Comment (tomponline) [reply bin/build-cache-lxd:139]: True, but I'm paranoid :)

**Pull request merged**